main/common/aperture.js

```diff
@@ -50,7 +50,7 @@ const callPlugins = async method => Promise.all(recordingPlugins.map(async ({plu
         })
       );
     } catch (error) {
-      showError(error);
+      showError(error, {title: `Something went wrong while using “${plugin.name}”`});
```

Comment:
Is `plugin.name` `kap-foo` or just `foo`. If just `foo`, I think this should be:
```suggestion
      showError(error, {title: `Something went wrong while using the plugin “${plugin.name}”`});
```

Reply:
It's `kap-foo` `plugin.prettyName` is `foo`. I can use either

Reply:
Use `plugin.prettyName`